boot/libs.ml

```diff
@@ -1,3 +1,5 @@
+let executables = [ "main" ]
+
```

Comment:
This change looks to be unrelated?

Reply:
Yes, I know the problem :-) Usually I can get this fixed when I rebootstrap from `main` with `make bootstrap`.